backends/graphics/openglsdl/openglsdl-graphics.cpp

```diff
@@ -335,6 +335,29 @@ void OpenGLSdlGraphicsManager::updateScreen() {
 	OpenGLGraphicsManager::updateScreen();
 }
 
+void OpenGLSdlGraphicsManager::lockWindowReSize() {
```

Comment:
These two functions (lock and unlock) don't bring much as you are only calling them at one place.